src/KM_Music.pas

```diff
@@ -423,10 +430,12 @@ procedure TMusicLib.UnfadeMusic;
   ZPlayer.GetPosition(StartTime);
   EndTime.ms := StartTime.ms + FADE_TIME;
   ZPlayer.GetPlayerVolume(Left, Right); //Start fade from the current volume
+  Sleep(25); // Sleep for 25 ms, give the stream time to buffer a few bytes
```

Comment:
Does this sleep(25) happens within main game loop, effectively suspending the whole game by 50ms?